### ISSUE TITLE
login_attempt problem on startup

### ISSUE THREAD

**Issue by BenoitRoux0** [our-transcendence/auth_service#26]
```shell
auth-service-1       | Traceback (most recent call last):
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py", line 103, in _execute
auth-service-1       |     return self.cursor.execute(sql)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/psycopg/cursor.py", line 732, in execute
auth-service-1       |     raise ex.with_traceback(None)
auth-service-1       | psycopg.errors.NotNullViolation: column "login_attempt" of relation "login_user" contains null values
auth-service-1       | 
auth-service-1       | The above exception was the direct cause of the following exception:
auth-service-1       | 
auth-service-1       | Traceback (most recent call last):
auth-service-1       |   File "/auth_service/auth/manage.py", line 22, in <module>
auth-service-1       |     main()
auth-service-1       |   File "/auth_service/auth/manage.py", line 18, in main
auth-service-1       |     execute_from_command_line(sys.argv)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
auth-service-1       |     utility.execute()
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
auth-service-1       |     self.fetch_command(subcommand).run_from_argv(self.argv)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/core/management/base.py", line 413, in run_from_argv
auth-service-1       |     self.execute(*args, **cmd_options)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/core/management/base.py", line 459, in execute
auth-service-1       |     output = self.handle(*args, **options)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/core/management/base.py", line 107, in wrapper
auth-service-1       |     res = handle_func(*args, **kwargs)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/core/management/commands/migrate.py", line 356, in handle
auth-service-1       |     post_migrate_state = executor.migrate(
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/migrations/executor.py", line 135, in migrate
auth-service-1       |     state = self._migrate_all_forwards(
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/migrations/executor.py", line 167, in _migrate_all_forwards
auth-service-1       |     state = self.apply_migration(
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/migrations/executor.py", line 252, in apply_migration
auth-service-1       |     state = migration.apply(state, schema_editor)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/migrations/migration.py", line 132, in apply
auth-service-1       |     operation.database_forwards(
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/migrations/operations/fields.py", line 108, in database_forwards
auth-service-1       |     schema_editor.add_field(
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/base/schema.py", line 755, in add_field
auth-service-1       |     self.execute(sql, params or None)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/postgresql/schema.py", line 45, in execute
auth-service-1       |     return super().execute(sql, params)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/base/schema.py", line 201, in execute
auth-service-1       |     cursor.execute(sql, params)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py", line 122, in execute
auth-service-1       |     return super().execute(sql, params)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py", line 79, in execute
auth-service-1       |     return self._execute_with_wrappers(
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
auth-service-1       |     return executor(sql, params, many, context)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py", line 100, in _execute
auth-service-1       |     with self.db.wrap_database_errors:
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/utils.py", line 91, in __exit__
auth-service-1       |     raise dj_exc_value.with_traceback(traceback) from exc_value
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py", line 103, in _execute
auth-service-1       |     return self.cursor.execute(sql)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/psycopg/cursor.py", line 732, in execute
auth-service-1       |     raise ex.with_traceback(None)
auth-service-1       | django.db.utils.IntegrityError: column "login_attempt" of relation "login_user" contains null values
auth-service-1       | [2024-04-29 08:05:42 +0000] [29] [INFO] Starting gunicorn 21.2.0
auth-service-1       | [2024-04-29 08:05:42 +0000] [29] [INFO] Listening at: http://0.0.0.0:4444 (29)
auth-service-1       | [2024-04-29 08:05:42 +0000] [29] [INFO] Using worker: sync
auth-service-1       | [2024-04-29 08:05:42 +0000] [30] [INFO] Booting worker with pid: 30
```

**Comment by BenoitRoux0** [comment]
unusable
```shell
auth-service-1       | Internal Server Error: /login/
auth-service-1       | Traceback (most recent call last):
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py", line 105, in _execute
auth-service-1       |     return self.cursor.execute(sql, params)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/psycopg/cursor.py", line 732, in execute
auth-service-1       |     raise ex.with_traceback(None)
auth-service-1       | psycopg.errors.UndefinedColumn: column login_user.totp_key does not exist
auth-service-1       | LINE 1: ...ogin_user"."picture", "login_user"."jwt_emitted", "login_use...
auth-service-1       |                                                              ^
auth-service-1       | 
auth-service-1       | The above exception was the direct cause of the following exception:
auth-service-1       | 
auth-service-1       | Traceback (most recent call last):
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
auth-service-1       |     response = get_response(request)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
auth-service-1       |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/views/decorators/csrf.py", line 65, in _view_wrapper
auth-service-1       |     return view_func(request, *args, **kwargs)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/views/decorators/http.py", line 64, in inner
auth-service-1       |     return func(request, *args, **kwargs)
auth-service-1       |   File "/auth_service/auth/login/endpoint.py", line 89, in login_endpoint
auth-service-1       |     user: User = User.objects.get(login=login)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/models/manager.py", line 87, in manager_method
auth-service-1       |     return getattr(self.get_queryset(), name)(*args, **kwargs)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/models/query.py", line 645, in get
auth-service-1       |     num = len(clone)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/models/query.py", line 382, in __len__
auth-service-1       |     self._fetch_all()
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/models/query.py", line 1928, in _fetch_all
auth-service-1       |     self._result_cache = list(self._iterable_class(self))
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/models/query.py", line 91, in __iter__
auth-service-1       |     results = compiler.execute_sql(
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
auth-service-1       |     cursor.execute(sql, params)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py", line 122, in execute
auth-service-1       |     return super().execute(sql, params)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py", line 79, in execute
auth-service-1       |     return self._execute_with_wrappers(
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
auth-service-1       |     return executor(sql, params, many, context)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py", line 100, in _execute
auth-service-1       |     with self.db.wrap_database_errors:
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/utils.py", line 91, in __exit__
auth-service-1       |     raise dj_exc_value.with_traceback(traceback) from exc_value
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/django/db/backends/utils.py", line 105, in _execute
auth-service-1       |     return self.cursor.execute(sql, params)
auth-service-1       |   File "/usr/local/lib/python3.10/site-packages/psycopg/cursor.py", line 732, in execute
auth-service-1       |     raise ex.with_traceback(None)
auth-service-1       | django.db.utils.ProgrammingError: column login_user.totp_key does not exist
auth-service-1       | LINE 1: ...ogin_user"."picture", "login_user"."jwt_emitted", "login_use...
```

**Comment by gd-harco** [comment]
fixed in #25
